src/main.rs

```diff
@@ -0,0 +1,62 @@
+extern crate clap;
+use clap::{App, Arg};
+use libwallet::{
+     self,
+     sr25519::{Pair, Public},
+     Pair as _, SimpleVault, Wallet,
+};
+use sp_core::crypto::{Ss58AddressFormat, Ss58Codec};
+
+#[async_std::main]
+async fn main() {
+     let matches = App::new("Wallet Generator")
+          .version("0.1.0")
+          .author("Daniel Olano <me@olanod.com>")
+          .about("Generates Wallet Account")
+          .arg(Arg::with_name("seed")
+               .short("s")
+               .long("from-seed")
+               .value_name("MNEMONIC")
+               .help("Generates a wallet address from mnemonic."))
+          .arg(Arg::with_name("network")
+               .short("n")
+               .long("network")
+               .value_name("NETWORK")
+               .help("Formats the address to specified network."))
+          .get_matches();
+
+     let pub_address = get_pub_address(matches.value_of("seed")).await;
+     let network: &str = matches.value_of("network").unwrap_or("");
+
+     let address: String = pub_address.to_ss58check_with_version(get_network_format(network));
```

Comment:
You don't need to do your own parsing since the type already supports parsing from strings, also if it fails we could just use the default which should be the substrate format 😉 

I think something like this should work
```suggestion
     let address: String = pub_address.to_ss58check_with_version(network.parse().unwrap_or_else(Default::default));
```